README.md

```diff
@@ -53,17 +53,19 @@ For B2B examples components please see:
 ### Usage
 
 <!-- usage -->
+
 ```sh-session
 $ npm install -g @salesforce/commerce
 $ sfdx COMMAND
 running command...
 $ sfdx (-v|--version|version)
-@salesforce/commerce/238.0.11 darwin-x64 node-v16.13.2
+@salesforce/commerce/238.0.12 darwin-x64 node-v12.16.3
```

Comment:
Why 12 instead of 16?

Reply:
I think this is auto-generated based on the node.js it is running